src/components/TabGroup.tsx

```diff
@@ -51,15 +53,32 @@ export function TabGroup(props: Props) {
         ))}
       </div>
 
-      <div className="my-4">
+      <div
+        className={classNames({
+          flex: props.height === 'full',
+          'flex-1': props.height === 'full',
+          'my-4': props.height !== 'full',
+        })}
+      >
         {[...props.children].map(
           (element, index) =>
             React.isValidElement(element) &&
             React.cloneElement(element, {
               key: index,
               // eslint-disable-next-line @typescript-eslint/ban-ts-comment
               // @ts-ignore
-              style: { display: currentIndex === index ? 'block' : 'none' },
+              style: {
+                display:
+                  currentIndex === index
+                    ? `${props.height === 'full' ? 'flex' : 'block'}`
+                    : 'none',
+                ...(props.height === 'full' && {
+                  flexDirection: 'column',
+                }),
+                ...(props.height === 'full' && {
+                  flexGrow: 1,
+                }),
+              },
```

Comment:
Is it possible to style these with Tailwind? The classes you are looking for are: flex, block, flex-1, flex-column

Reply:
Sure, by passing the style in the className instead of the style property we can achieve this. So this is refactored!